entries/m/migros.ch.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Migros": {
+    "passwordless": "allowed",
+    "mfa": "allowed",
+    "categories": "retail",
+    "documentation": "https://account.migros.ch/help#faqPasskeyPrio",
+    "recovery": "https://account.migros.ch/help#faqPasskeyPrio",
+    "regions": [
+      "at",
```

Comment:
I don't see this region listed in the [2FA Directory PR](https://github.com/2factorauth/twofactorauth/pull/8255).